src/app/core/map/canvas-container.directive.ts

```diff
@@ -31,20 +31,22 @@ export class CanvasContainerDirective implements AfterViewInit {
     // autoStart: false
   };
 
-  constructor(private el: ElementRef, private zone: NgZone) { 
-    this.element = <HTMLDivElement> el.nativeElement;
+  constructor(private el: ElementRef, private zone: NgZone) {
+    this.element = el.nativeElement as HTMLDivElement;
 
-    const options = Object.assign({width: this.element.clientWidth, height: this.element.clientHeight},
-                    this.applicationOptions);
+    const options = Object.assign({ width: this.element.clientWidth, height: this.element.clientHeight },
+      this.applicationOptions);
 
     this.zone.runOutsideAngular(() => {
       // prevents pixi ticker to clash with zone
       this.app = new PIXI.Application(options);
-      
+
       // prevents mouse zoom on document
       window.addEventListener('wheel', e => {
-        e.preventDefault();
-      }, {passive: false});
+        if (!(e.currentTarget as HTMLElement)?.closest('app-initiative-list')) {
```

Comment:
this line is causing an error in latest safari and chrome:
`zone-evergreen.js:171 Uncaught TypeError: _a.closest is not a function`
looks like the `e.currentTarget` is a Window object and not a HTMLElement, so closest is not defined

Reply:
hmm, looks like it's working fine when adding event to canvas element instead of window.